File: app/src/debug/java/nz/co/chrisdrake/events/data/api/model/MockLocationResponse.java
package nz.co.chrisdrake.events.data.api.model;

import java.util.Collections;

public final class MockLocationResponse {
    public static final LocationResource UNITED_STATES =
        LocationResource.create(Collections.singletonList(MockLocations.UNITED_STATES));

    private MockLocationResponse() {
        throw new AssertionError("Non-instantiable.");
    }
}

File: app/src/debug/java/nz/co/chrisdrake/events/data/api/model/MockLocationResponse.kt
package nz.co.chrisdrake.events.data.api.model

object MockLocationResponse {
    @JvmField val UNITED_STATES = LocationResource(locations = listOf(Locations.UNITED_STATES))

    private object Locations {
        val SAN_FRANCISCO = Location(
                id = 101,
                name = "San Francisco")

        val UNITED_STATES = Location(
                id = 100,
                name = "United States",
                children = LocationChildResource(children = listOf(SAN_FRANCISCO)))
    }
}
